src/libraries/System.Private.CoreLib/src/System/Buffers/TlsOverPerCoreLockedStacksArrayPool.cs

```diff
@@ -211,7 +211,6 @@ public bool Trim()
             // Under high pressure, release all thread locals.
             if (pressure == Utilities.MemoryPressure.High)
             {
-#if !MONO // TODO https://github.com/mono/linker/issues/2181: Remove !MONO ifdefs in this method once  is fixed
```

Comment:
Not sure if it was intentional or not, but there's also the ifdef below:
https://github.com/dotnet/runtime/pull/56813/files#diff-42062b57ac1000ebcd8446e0f1c2820574c729bd5b778da786241592892cec30L273